bundles/EcommerceFrameworkBundle/EventListener/Frontend/TrackingCodeFlashMessageListener.php

```diff
@@ -90,14 +92,30 @@ public function onKernelRequest(RequestEvent $event)
         }
     }
 
+    /**
+     * @return SessionInterface
+     */
+    private function getSession()
+    {
+        try {
+            return $this->requestStack->getSession();
+        } catch (SessionNotFoundException $e) {
+        }
+
```

Comment:
we should not go for a fallback here but return null

Reply:
fixed here https://github.com/pimcore/pimcore/pull/12746/commits/a5c037bad3ad318097cd68c6bd3dfe294e72c17d